docs/UserGuide.md

```diff
@@ -241,26 +253,117 @@ An example of an error message is shown below.
 
 ![img.png](DeleteCommandError.png)
 
+### Add remark to a company: `remark` ###
+If you have information not within the fields,
+use the `remark` command to input the additional details.
+
+**Format:** `remark INDEX re/REMARK`
+
+>`INDEX`: Adds remark of the company at INDEX number shown in the displayed company list. </br>
+>`[re/REMARK]` : type prefix `re/` followed by the remark you want to add </br>
```

Comment:
Don't use brackets around re/REMARK? I think we will put a note that [] means that the fields are optional, in this case it isn't